README.md

```diff
@@ -3,41 +3,59 @@ Repository faking an add-ons server for [@widelands](https://github.com/wideland
 
+[Informations how to create add-ons can be found here.](https://github.com/widelands/widelands/blob/master/doc/sphinx/source/add-ons.rst)
+
+If you do not wish to create a pull request for a new add-on here, it is also possible to submit add-ons on the forum: https://www.widelands.org/forum/topic/5073
+
+### Prerequisites
+
+Before adding add-ons to this repository be sure a java development kit (JDK) is installed to compile the UpdateList.java script.
+
+After the installation of the JDK run `javac UpdateList.java` in the base folder.
+
 ### Adding a new add-on
 
-Just add it to the `addons/` folder, then run `java UpdateList`, then git add,commit,push.
+#### Addon
 
-### Updating an add-on
+Just add the folder `name_of_addon.wad` to the `addons/` folder.
 
-Just replace the add-on’s entry in the `addons/` folder with the new one, then run `java UpdateList`, then git add,commit,push.
+#### Screenshots
 
-### Updating screenshots
+If you want to provide some screenshots for your addon, add the screenshots to the subfolder `screenshots/name_of_addon.wad` and create a file called `descriptions`. Refer to the examples to get used to this file.
 
-When you updated any of the files in `screenshots/` or added new ones, just run `java UpdateList`, then git add,commit,push. Be sure to update the `descriptions` file beforehand.
+#### Apply changes
 
-### Updating translations
+After adding the addon and probably some screenshots, run `java UpdateList` inside the base folder, then git add,commit,push. Then create a pull request here.
 
-When you updated any of the files in `po/` or added new ones, just do git add,commit,push.
+### Updating an add-on
 
-A GitHub action automatically compiles the translations periodically by running `buildcats.sh`. **Do not** run this script manually. If you have to do it anyway for some reason, do this **only** on the master branch.
+Just replace the folder below `addons` with the new one.
+Same applies to screenshots: Just replace or add new ones. Be sure to update the `descriptions` file.
+
+After you have done run `java UpdateList`, then git add,commit,push.
 
 ### Verifying add-ons
 
-Run `java UpdateList '/cool_feature.wad'` (don’t forget the '/' before the add-on’s name!), then run `java UpdateList`, then git add,commit,push.
+Updating an add-on will automatically set it to Not Verified. It is not allowed to mark an add-on as verified during the update.
+To verify it run `java UpdateList '/cool_feature.wad'` (don’t forget the '/' before the add-on’s name!), then run `java UpdateList`, then git add,commit,push.
+
+### Uploader
 
-### Notes
+The name of the uploader of a new add-on is automatically set to `"Nordfriese"` – if you are not me, consider editing the list manually after running `buildcats.sh` or `java UpdateList` to replace it with your name. (Each add-on's uploader is listed directly below the author's name.)
 
```

Comment:
As far i understand running buildcats.sh isn't needed to get, yes what?

Where do i have to modify the uploader? I guess in the file called list, or list_2, or both?

Reply:
`buildcats.sh` is used for the translations updates, to sync the PO source files with the downloadable MO files. This script is now run automatically by a GitHub action every night, so it needs to be run manually only when I am doing some maintenance that messes up the repo structure.

`list_2` is the version parsed by the current Widelands master; `list` has a slightly different format used by "older" Widelands versions. The uploader name should be edited in both files.